modules/cli-logging-in.adoc

```diff
@@ -10,7 +10,14 @@ You can log in to the OpenShift CLI (`oc`) to access and manage your cluster.
 
 .Prerequisites
 
-* You must have access to an {product-title} cluster.
+* You must have access to 
+ifndef::openshift-rosa[]
+an {product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+a ROSA
+endif::openshift-rosa[]
+cluster.
 * You must have installed the OpenShift CLI (`oc`).
```

Comment:
```suggestion
* The {oc-first} is installed.
```
Out of scope, but we don't need to tell every user to install `oc`, it might already be there.